apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryRow.cs

```diff
@@ -183,7 +183,8 @@ private static T[] ConvertArray<T>(JArray array, Func<object, T> converter)
             {
                 var field = fields[i];
                 var token = values[i]["v"];
-                ret[field.Name] = ConvertSingleValue(token.Type == JTokenType.String ? (string)token : (object)token, field);
+                ret[field.Name] = token.Type == JTokenType.Null ? null
```

Comment:
Have worked out a way of doing it, actually - and generally feel a bit better about it. PTAL.